core/src/main/resources/org/elasticsearch/tasks/task-results-index-mapping.json

```diff
@@ -0,0 +1,43 @@
+{
+  "result" : {
+    "dynamic" : "strict",
+    "properties" : {
+      "task" : {
+        "properties": {
+          "action": {
+            "type": "keyword"
+          },
+          "cancellable": {
+            "type": "boolean"
+          },
+          "id": {
```

Comment:
Maybe we don't need the `id` at all here? It is part of the document's id, right?